src/talondoc/sphinx/directives/command/__init__.py

```diff
@@ -42,4 +42,4 @@ def handle_signature(self, sig: str, signode: addnodes.desc_signature) -> str:
             return command.name
         except (UnknownReference, AmbiguousSignature) as e:
             _LOGGER.error(f"talon:command: {e}")
-            raise ValueError(e)
+            raise ValueError(e) from e
```

Comment:
What does this change do?

Reply:
It is a way of flagging an exception as direct consequence of another https://docs.python.org/3/tutorial/errors.html#exception-chaining